spikeextractors/extraction_tools.py

```diff
@@ -252,7 +252,8 @@ def read_binary(file, numchan, dtype, time_axis=0, offset=0):
     return samples
 
 
-def write_to_binary_dat_format(recording, save_path, time_axis=0, dtype=None, chunk_size=None, chunk_mb=500):
+def write_to_binary_dat_format(recording, save_path=None, file_handle=None,
+                               time_axis=0, dtype=None, chunk_size=None, chunk_mb=500):
```

Comment:
**chunk_mb** very very cool. I will propagate into sorters.

Reply:
by default it's 500mb. I think it's an ok value :)